extensions/currency-exchange/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "currency-exchange",
+  "title": "Currency Exchange",
+  "description": "Simple Currency Exchange",
+  "icon": "currency.png",
+  "author": "xeric",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Currency Exchange",
+      "description": "Calculate with Exchange Rate",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "api_key",
+      "type": "password",
+      "required": true,
+      "title": "API Key",
+      "description": "You can get API Key at https://app.exchangerate-api.com/dashboard",
+      "placeholder": "Exchange Rate API Key"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.35.0",
+    "node-fetch": "^3.0.0",
+    "run-applescript": "^6.0.0",
+    "sound-play": "^1.1.0"
```

Comment:
Are they even required?

Reply:
Great catch! I forget to clean up useless dependencies, will update ASAP 👍